src/github.ts

```diff
@@ -1034,6 +1034,31 @@ export class GitHub {
       return path;
     });
   }
+
+  async findFilesByExtension(
+    extension: string,
+    prefix?: string
+  ): Promise<string[]> {
+    let q = `extension:${extension}+repo:${this.owner}/${this.repo}`;
+    if (prefix) {
+      prefix = prefix.replace(/[/\\]$/, '');
```

Comment:
IF we find ourselves doing this too many more times, it might be worth pulling into a helper, `removeLeadingSlash`?

Could just be in the `github.js` class I suppose.